.github/workflows/kcp-upgrade.yaml

```diff
@@ -19,10 +19,11 @@ jobs:
       - name: Check if a new kcp version exists and update the files if it does
         id: fetch-kcp-version
         run: |
-          sh ./images/kcp-upgrade/upgrade.sh
-          echo "::set-output name=NEW_KCP_VERSION::$(curl -s https://api.github.com/repos/kcp-dev/kcp/releases/latest | yq '.tag_name' | sed 's/v//')"
-          echo "::set-output name=NEW_KCP_VERSION_FOUND::true"
-        continue-on-error: true
+          ./images/kcp-upgrade/upgrade.sh
+          if [ -e "/tmp/kcp-upgrade.txt" ]; do
```

Comment:
I've not seen this syntax before - if->do->done. Shouldn't it be if->then->fi?

Reply:
Fixed